sql/30_station.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE `station` (
+ `id`                       VARCHAR(35) NOT NULL,
+ `lat`                      DOUBLE(18,14) NOT NULL,
+ `lon`                      DOUBLE(18,14) NOT NULL,
+ `name`                     VARCHAR(128) NOT NULL,
+ `cell_id`                  BIGINT UNSIGNED NOT NULL,
```

Comment:
All my Cell ID's are negative numbers on the East Coast of the US so Unsigned for cell_id doesn't work and resulted in 
insert station: Error 1264 (22003): Out of range value for column 'cell_id' at row 1

Reply:
does it work if you adjust the table by hand?

Reply:
Yes I manually changed the table to no be unsigned and it is now putting them in the table properly.

Reply:
thanks, fixed... now it's aligned with the other `cell_id` columns, was a copy-paste error :)